src/index.js

```diff
@@ -1,81 +1,99 @@
-import React, { useRef, useState, useEffect } from 'react'
-import PropTypes from 'prop-types'
+import React, { useRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
 
-const Collapsable = ({
+const Collapsible = ({
```

Comment:
Collapsable was intentional, you can spell it both ways and there is already a react-collapsible on NPM.

Reply:
I will change this back.